.gitignore

```diff
@@ -0,0 +1,2 @@
+.env
+vendor/
```

Comment:
dependencies and other files should go into the plugins folder, eg

plugins/authsaml/vendor

and then referenced from there. Don't put any code below the plugins folder.

Reply:
This has been addressed